README.md

```diff
@@ -89,6 +89,20 @@ The modem definition a 3 item tuple in this format
 `{modem_name, provider, modem_module}` where the modem module is module that
 implements the `VintageNetLTE.Modem` behaviour.
 
+If you are implementing a modem that does not need to use `VintageNetLTE` for
+provider information you can configure this like so:
+
+```elixir
+config :vintage_net_lte,
+  extra_modems: [
+    {"UBlock", MyModem}
```

Comment:
I think we should encourage a naming convention in the strings so that they'll list nicely. Like `{"ublox - SARA-R999", MyModem}`. Fwiw, that ublox modem doesn't exist, so it's probably a decent example.